File: app/src/main/java/com/bq/zowi/interactors/MeasureZowiBatteryLevelInteractor.java
package com.bq.zowi.interactors;

import rx.Single;

/* JADX INFO: loaded from: classes.dex */
public interface MeasureZowiBatteryLevelInteractor {
    public static final float BATTERY_LEVEL_LOWER_THRESHOLD = 50.0f;

    Single<Boolean> manageZowiBatteryLevel(float f);

    Single<Boolean> measureAndManageZowiBatteryLevel();
}

File: zowi-core/src/main/kotlin/com/bq/zowi/usecases/MeasureZowiBatteryLevelInteractor.kt
package com.bq.zowi.usecases

import io.reactivex.Single

interface MeasureZowiBatteryLevelInteractor {
    companion object {
        const val BATTERY_LEVEL_LOWER_THRESHOLD = 50.0f
    }

    fun manageZowiBatteryLevel(batteryLevel: Float): Single<Boolean>
    fun measureAndManageZowiBatteryLevel(): Single<Boolean>
}
